unfold_lineageos.sh

```diff
@@ -4,7 +4,7 @@ LOCAL_PATH=$(pwd)
 
```

Comment:
Just giving a small heads-up... I doubt I'd be able to handle both AOSP and lineage for each device in CI. I'd just build both of them only for release and have a parameterized jenkins job that would build a lineage based image from any commit.

Reply:
No problem, I tested it